pkg/lib/config/sms_gateway_env.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+type SMSGatewayEnvironmentTwilioCredentials struct {
+	AccountSID          string `envconfig:"ACCOUNT_SID"`
+	AuthToken           string `envconfig:"AUTH_TOKEN"`
+	MessagingServiceSID string `envconfig:"MESSAGING_SERVICE_SID"`
+}
+
+type SMSGatewayEnvironmentNexmoCredentials struct {
+	APIKey    string `envconfig:"API_KEY"`
+	APISecret string `envconfig:"API_SECRET"`
+}
+
+type SMSGatewayEnvironmentCustomSMSProviderConfig struct {
+	URL     string          `envconfig:"URL"`
+	Timeout DurationSeconds `envconfig:"TIMEOUT"`
+}
+
+type SMSGatewayEnvironmentUseConfigFrom string
```

Comment:
Rename SMSGatewayEnvironmentUseConfigFrom to SMSGatewayEnvironmentDefaultUseConfigFrom